docs/getting-started/x-plat/sample/src/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Linq;
+using System.Text;
+
+namespace ConsoleApp
+{
+    public class Program
+    {
+        public void Main(string[] args)
+        {
+            using (var context = new BloggingContext())
+            {
+                var newPost = new Post { };
+
+                Console.Write("New post title >");
+
+                newPost.Title = Console.ReadLine();
+
+                Console.WriteLine("Post content >\n(Press ENTER twice to complete");
+
+                var sb = new StringBuilder();
+                string line = Console.ReadLine();
+                do
+                {
+                    sb.AppendLine(line);
+                }
+                while (!string.IsNullOrEmpty(line));
+
+                newPost.Content = sb.ToString();
+
+                context.Posts.Add(newPost);
+                context.SaveChanges();
+
+                Console.WriteLine("New post '${ newPost.Title }' saved!");
+            }
+        }
+    }
+}
```

Comment:
This code block should be consistent with the other walkthroughs (unless there is a need to change it for OSX).

```
            using (var db = new BloggingContext())
            {
                db.Blogs.Add(new Blog { Url = "http://blogs.msdn.com/adonet" });
                var count = db.SaveChanges();
                Console.WriteLine("{0} records saved to database", count);

                Console.WriteLine();
                Console.WriteLine("All blogs in database:");
                foreach (var blog in db.Blogs)
                {
                    Console.WriteLine(" - {0}", blog.Url);
                }
            }
```